vignettes/v1-2020-2021-mitma-data-codebook.qmd

```diff
@@ -0,0 +1,228 @@
+---
+title: "Codebook and cookbook for v1 (2020-2021) Spanish mobility data"
+vignette: >
+  %\VignetteIndexEntry{Codebook and cookbook for v1 (2020-2021) Spanish mobility data}
+  %\VignetteEngine{quarto::html}
+  %\VignetteEncoding{UTF-8}
+bibliography: references.bib
+format:
+  html:
+    toc: true
+    toc-depth: 2
+    code-overflow: wrap
+execute: 
+  eval: false
+---
+
+You can view this vignette any time by running:
+
+```{r}
+spanishoddata::spod_codebook(ver = 1)
+```
+
+The mobility data v1 (2020-2021) was originally released by the [Ministry of Transport and Sustainable Mobility (MITMA)](https://www.transportes.gob.es/) [@mitma_data_v1]. The dataset is produced by [Nommon](https://www.nommon.es/){target="_blank"} using the raw data from [Orange España](https://www.orange.es/){target="_blank"}. Even though the raw data is only from one mobile phone operator, the resulting flows and and other counts of number of individuals in the data set are already resampled to be representative of the total population of Spain. The tables in the data set provide hourly flows between zones across Spain for every day of the observation period (2020-02-14 to 2021-05-09) and the number of trips per person made. This document will introduce you to the available data and provide brief code snippets on how to access it using the `spanishoddata` R package.
+
+Key sources for this codebook/cookbook include:
+
+-   [original data collection methodology in Spanish](https://cdn.mitma.gob.es/portal-web-drupal/covid-19/bigdata/mitma_-_estudio_movilidad_covid-19_informe_metodologico_v3.pdf){target="_blank"} + [automatically translated English version of methodology](https://robinlovelace.github.io/spanishoddata/assets/codebooks/mitma_-_estudio_movilidad_covid-19_informe_metodologico_v3_en.pdf){target="_blank"}
+
+-   [original data codebook in Spanish](https://opendata-movilidad.mitma.es/README%20-%20formato%20ficheros%20movilidad%20MITMA%2020201228.pdf){target="_blank"} + [automatically translated English version of codebook](https://robinlovelace.github.io/spanishoddata/assets/codebooks/README_-_formato_ficheros_movilidad_MITMA_20201228_en.pdf){target="_blank"}
+
+-   [original data download page](https://www.transportes.gob.es/ministerio/proyectos-singulares/estudios-de-movilidad-con-big-data/estudios-de-movilidad-anteriores/covid-19/opendata-movilidad){target="_blank"}
+
+-   [original data license](https://opendata-movilidad.mitma.es/LICENCIA%20de%20datos%20abiertos%20del%20MITMA%2020201203.pdf){target="_blank"}, and the (5) [homepage of the open mobility data project of the Ministry of Transport and Sustainable Mobility of Spain](https://www.transportes.gob.es/ministerio/proyectos-singulares/estudio-de-movilidad-con-big-data){target="_blank"}.
```

Comment:
Ahhh, the acronym of the Ministry is so problematic. They changed the name and the website address several times of the past two years) The data license clearly mentions MITMA and urges the data users to cite it as MITMA open data. I agree we can include the new name too. I will rephrase.
By the way, would you say it is still correct to also refer to them as Secretary of State for Transport, Mobility and Urban Agenda (Secretaria de Estado de Transportes, Movilidad y Agenda Urbana)... This is what they have in the methodology docs. Is that the same as Ministry...? I'm confused here.

Reply:
I know. It is all Pedro Sanchez fault! :) He had some unpopular periods and reshaped some departments to make the government appealing and efficient. This is the way these departments look in the update 18/07/2024.
![Screenshot from 2024-08-26 15-29-16](https://github.com/user-attachments/assets/30a1deb3-383c-490e-b209-0d41d2ff3d29). Perhaps it is worth it to the data provider how they prefer to be citted?  
The Secretary of State for Transport, Mobility and Urban Agenda (Secretaria de Estado de Transportes, Movilidad y Agenda Urbana) is also old. It does not exist in the screenshot above. However, it seems this Secretary started the project.

Reply:
I would say it makes sense to cite them according to the docs on the website and provide a link to the website. To me that makes sense and would still be traceable to the whichever name they may have in the future because of the link to the website.

Reply:
@e-kotov Great! It looks fine in the new version.

Reply:
@e-kotov Sorry, I said it looks fine, but now that I'm rereading the intro I was wondering whether it wouldn't be better if you removed the translation of the citation of the Secretaría de Estado de Transportes, Movilidad y Agenda Urbana in the citation. I do not think it is necessary (you do not translate the ministries) and that would make it easier to read and avoid those awkward brackets within brackets. Or perhaps remove the Spanish?